packages/popover2/src/popover2.tsx

```diff
@@ -308,6 +308,9 @@ export class Popover2<T> extends AbstractPureComponent2<Popover2Props<T>, IPopov
             : {
                   // CLICK needs only one handler
                   onClick: this.handleTargetClick,
+                  // For keyboard accessibility, we trigger the same behavior as a click event upon pressing enter
+                  onKeyDown: (event: React.KeyboardEvent<HTMLElement>) =>
+                      event.key === "Enter" && this.handleTargetClick(event),
```

Comment:
We use a util function called `Keys.isKeyboardClick` in other places to enable this kind of behavior. I suggest we do the same here, and make the popover open on either ENTER or SPACE. https://github.com/palantir/blueprint/search?q=Keys.isKeyboardClick